packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -162,8 +164,11 @@ pub trait Interval<T>: Sized {
     where
         T: PartialOrd,
     {
-        // Range:        [-----]   | ---)    | ---]    |    (--- |    [---
-        // Complement: --)     (-- |    [--- |    (--- | ---]    | ---)
+        // Examples   |       1     |    2    |    3    |    4    |    5
```

Comment:
Not important, and maintaining it might prove a bit annoying, but the `1` is off center
```suggestion
        // Examples   |      1      |    2    |    3    |    4    |    5
```